source/wp-content/themes/wporg-showcase-2022/src/site-post-template-wrap/index.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Block Name: Site Post Template Wrap
```

Comment:
Ha! I originally called it `Link Wrap` but since I wasn't certain I had time to implement an interface where the link **isn't** the post's permalink, I tried to think of something that would communicate it as something to be added within a `post-template` (although arguably I could have locked it to a `post-template` parent).

I like "Link Group". 👍